platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerView.swift

```diff
@@ -25,15 +25,18 @@ public struct WysiwygComposerView: UIViewRepresentable {
     public var replaceText: (NSAttributedString, NSRange, String) -> Bool
     public var select: (NSAttributedString, NSRange) -> Void
     public var didUpdateText: (UITextView) -> Void
+    public var textColor: UIColor
 
     public init(content: WysiwygComposerContent,
                 replaceText: @escaping (NSAttributedString, NSRange, String) -> Bool,
                 select: @escaping (NSAttributedString, NSRange) -> Void,
-                didUpdateText: @escaping (UITextView) -> Void) {
+                didUpdateText: @escaping (UITextView) -> Void,
+                textColor: UIColor = .label) {
```

Comment:
Probably makes sense to order this param before the closures so that we can use the trailing closure syntax?

Reply:
Absolutely, actually I was wondering if we even want the parameter in init, we could also just leave it as a public var and use the modifier I created for the view, to set it. What do you think?

Reply:
Works for me

Reply:
done!